### PR TITLE
Feature request: Signup form from the post layout

### PR DESCRIPTION
**Pull request by delisma**
---
name: Signup form
about: This a gerneral pull request to add a signup form to the newsletter in the post layout
title: 'Feature request: Signup form from the post layout'
labels: 'enhancement'
assignees: '@delisma'
---

### What does this MR do?

This PR addresses the feature request in issue #34. The main solution would be an email input field directly at the bottom of the blog post.

image

### General checklist

- [x] [Documentation](README.md) created/updated
- [x] Changelog entry added, if necessary
- [ ] Tests added for this feature/bug
- [x] Conforms to the [style guides](https://www.canada.ca/en/government/about/design-system.html)

### Related issues
#34

### CODE REVIEW

**Review by arianna-merritt — APPROVED** [review]

**Comment by arianna-merritt** [comment]
@LPipes4 - Thoughts?

**Comment by delisma** [comment]
en: https://deploy-preview-35--blog-tbs.netlify.app/2021/05/10/codesign-with-deciders
fr: https://deploy-preview-35--blogue-sct.netlify.app/2021/05/10/participation-approbateurs-conception

File: _layouts/post.html [hunk 1]
```diff
@@ -4,17 +4,93 @@
...
+											<label for="mce-EMAILTYPE-1">{{ locales.emailtype2[ page.lang ] }}</label> <small>{{ locales.emailtype-cmmt[ page.lang ] }}</small>
+										</li>
+									</ul>
+								</fieldset>
+								<div class="mc-field-group">
+									<div class="input-group">
+										<label class="wb-inv" for="mce-EMAIL">{{ locales.mce-email[ page.lang ] }}</label>
+										<input type="email" value="" name="EMAIL" class="form-control required email" id="mce-EMAIL" placeholder="{{ locales.mce-email[ page.lang ] }}">
+										<span class="input-group-btn">
+											<button type="submit" name="subscribe" id="mc-embedded-subscribe" class="btn btn-success nowrap">{{ locales.subscribe[ page.lang ] }}</button>
```
Comment (arianna-merritt) [inline _layouts/post.html:41]: @delisma Please make sure it still links to Mailchimp

**Review by arianna-merritt — CHANGES_REQUESTED** [review]

File: _layouts/post.html (hunk 1 above)
Comment (delisma) [reply _layouts/post.html:41]: Yes the form does subscribe us in the DTO newsletter but the confirmation page has a button `contibue to our website` the leads to the wrong URL. The URL is https://canada-ca.github.io/ which brings people to a splash page that looks unstyled and still not bringing people to the right place.

image
image
These links go to https://canada-ca.github.io/blog-home-test-index.html and https://canada-ca.github.io/

Comment (delisma) [reply _layouts/post.html:41]: Made a [PR](https://github.com/canada-ca/canada-ca.github.io/pull/53) in github/canada-ca/canada.github.io repo, and changed the splash page.

**Review by arianna-merritt — APPROVED** [review]

**Pull request merged**